openedx/features/partners/give2asia/views.py

```diff
@@ -164,6 +165,15 @@ def create_account_with_params_custom(request, params, partner):
     # Perform operations that are non-critical parts of account creation
     preferences_api.set_user_preference(user, LANGUAGE_KEY, get_language())
 
+    try:
+        partner_communities = PartnerCommunity.objects.filter(partner=partner)
+        for community in partner_communities:
+            task_join_group_on_nodebb.delay(
+                category_id=community.category_id, username=user.username)
+    except Exception as err:
```

Comment:
Let's please remove try catch because neither task not filter will throw any exception. Also how about if we just add a log after calling task, then we have made a task for user bla bla bla....

Reply:
Done